### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by forerosantiago**
Hello, I have translated the plugin to the Spanish language, followed the instructions given in #182 

Please let me know if there are any issues or problems with my contribution.

Thank you and Happy Coding!

### CODE REVIEW

File: src/main/resources/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 # Plugin language. Empty: system language.
-# Available: en-US (default, fallback), fr-FR, ru-RU, de-DE, zh-CN, ja-JP.
+# Available: en-US (default, fallback), fr-FR, ru-RU, de-DE, zh-CN, ja-JP, es-US.
```
Comment (Vlammar) [inline src/main/resources/config.yml]: Is there a real difference between es-US and es-ES ?  This should be es-ES right?

**Review by Vlammar — CHANGES_REQUESTED** [review]
Thank you for the translation :)

File: src/main/resources/config.yml (hunk 1 above)
Comment (forerosantiago) [reply src/main/resources/config.yml]: Oh yeah, I missed that. It is es-ES. Sorry.

Comment (Vlammar) [reply src/main/resources/config.yml]: No problem, I can edit it right now if you want me to

**Review by Vlammar — APPROVED** [review]
That's good now, thank you very much 👍

**Comment by forerosantiago** [comment]
Always happy to help!

**Pull request merged**